dev/snippets/config/templates/README.md

```diff
@@ -77,29 +82,34 @@ follows:
   `build()` method, and any state variables. It also has an `imports`
   section to import additional packages. Please only import things that are part
   of flutter or part of default dependencies for a `flutter create` project.
-  It creates a `WidgetsApp` around the child stateful widget.
+  
+- [`stateful_widget_ticker`](stateful_widget_ticker.tmpl) : Identical to the
+  `stateful_widget` template, with the addition of the `TickerProviderStateMixin`
+  class, enabling easy generation of animated samples.
 
 - [`stateless_widget`](stateless_widget.tmpl) : Identical to the
   `stateful_widget` template, except that the default code block is
   inserted as a method (which should be the `build` method) in a
   `StatelessWidget`. The `@override` before the build method is added by
   the template, so must be omitted from the sample code.
 
-- [`stateful_widget_material`](stateful_widget_material.tmpl) : Similar to
-  `stateful_widget`, except that it imports the material library, and uses
-  a `MaterialApp` instead of `WidgetsApp`.
+### MaterialApp Templates
 
-- [`stateless_widget_material`](stateless_widget_material.tmpl) : Similar to
-  `stateless_widget`, except that it imports the material library, and uses
-  a `MaterialApp` instead of `WidgetsApp`.
+These templates follow the same conventions as the `WidgetsApp` templates above, but use a
+`MaterialApp` instead. These templates also import the  material library.
```

Comment:
Just being pedantic: these templates import the material library.

The updates look good.